### DISCUSSION TITLE
Spectrophotometer absorbance values to RGB space

### DISCUSSION THREAD

**Discussion by chloefouilloux** [thomased/colRverse#7] [Q&A] [answered]
Hello! 

I am hoping to use your awesome package to covert my spectrophotometer data (measured in absorbance) to the RGB space. In the vignettes the data is coded as %Reflectance, and from what I understand the "spec2rgb" function works only with %reflectance data, is that correct?

Can you guys point me in the right direction of how to convert my transmission data to the RGB colour space? Any help would be very warmly welcome. 

Chloe

**Comment by thomased** [comment, marked as answer]
Emailed, but just posting my answer here too in case! 

Yeah you’re right that the package (including spec2rgb) is understandably centred on reflectance. So for spec2rgb() to work I guess you could just make some assumptions and convert your absorbance to reflectance and go from there? In the simplest possible case that’d just be Reflectance = 1 – (Transmittance + Absorbance). And if you assume there’s no transmittance, then just subtract your % absorbances from 1 to get a reflectance spectrum (i.e. any light that isn’t absorbed, is reflected).
 
Again, that’s a simple case, but might be all you need depending on your question(s). Hope that helps anyway - just shout if you hit any snags.
